### ISSUE TITLE
Multiranger deck gives odd values when out of range

### ISSUE THREAD

**Issue by krichardsson** [bitcraze/crazyflie-firmware#874]
Reported in https://forum.bitcraze.io/viewtopic.php?f=6&t=5086

We have also seen odd behavior when the multiranger sensors are out of range. A user probably expects some know value (or other mechanism) to be able to distinguish out of range from real distance measurements.

I think the Z-ranger behaves differently and reports a fixed distance when out of range. The sensor has a wide range of configurations and the Z-ranger is not set up in the same way as the multi ranger (short-range VS long-range).

**Comment by rafaelrgb** [comment]
We are also seeing strange behavior when the multiranger is out of range. Here is an experiment I made, where I moved the crazyflie from one wall (left sensor - green line) to the other (right sensor - blue line). The left sensor starts at a small value and increases as I move away from the wall. When I reach the maximum range (around 395 on the horizontal axis) there is a high increase in the noise and then the sensor starts to report a short range, which increases as I keep moving farther and farther. This behavior is mirrored by the opossite sensor.

![crazyflie_range](https://user-images.githubusercontent.com/22923579/143914385-bbf6782f-e795-456a-b1ba-1d290796e267.png)
